packages/nvidia_nat_opentelemetry/src/nat/plugins/opentelemetry/register.py

```
@@ -209,6 +210,49 @@ async def galileo_telemetry_exporter(config: GalileoTelemetryExporter, builder:
+        default="https://trace.wandb.ai/otel/v1/traces",
+    )
+    api_key: SerializableSecretStr = Field(description="The W&B API key",
+                                           default_factory=lambda: SerializableSecretStr(""))
+    project: str = Field(description="The W&B project name.")
+    entity: str = Field(description="The W&B username or team name.")
+
+
+@register_telemetry_exporter(config_type=WeaveOtelTelemetryExporter)
+async def weave_otel_telemetry_exporter(
+    config: WeaveOtelTelemetryExporter, builder: Builder,
+) -> AsyncGenerator["OTLPSpanAdapterExporter", None]:
```

Ruff (0.15.7)

[error] Undefined name `OTLPSpanAdapterExporter`

(F821)